### DISCUSSION TITLE
Rome have plans concerning monorepo ?

### DISCUSSION THREAD

**Discussion by c0ncentus** [rome/tools#4199] [Q&A]
hi,

I test lerna and nx, failed to achieve monorepo with **existing projects**  but have some issues  :
- cannot build/published on "get started" on monorepo typescript package
- have errors on tsconfig/webpack/babel on multi cra apps monorepo (i used the same config for all before and after ...)

I hate when it comes to solve babel, webpack issues (I remember testing runner don't compile because some intern tsCompile errors. It's mean **the test runner crash the app because overriding tsConfig rules**).

In this case test runner **should not** raise errors and override existing rule. 
The same with all layers like monorepo  etc ...  

 
I tried for :
- Typescript packages
- CRA apps and shared lib

### do Rome has/have/will planned monorepo features ?

**Comment by GabenGar** [comment]
It really depends on what expectations for "planning" you have.
The roadmap on the site shows Nov 2023 as a deadline for CSS linter/formatter, and all other points are linter/formatter too, so I assume the work on bundler won't start/is on hiatus at least until then. And whatever monorepo solution will not arrive before the bundler either.

**Comment by c0ncentus** [comment]
I expected "WIP that rome will do"

**Comment by milesj** [comment]
You should try moon: https://moonrepo.dev/moon
